### PR TITLE
postgresql_info: fix crash when extension version is 0.

### PR DESCRIPTION
**Pull request by Andersson007**
##### SUMMARY

Fixes https://github.com/ansible-collections/community.postgresql/issues/137
Fixes https://github.com/ansible-collections/community.postgresql/issues/136

postgresql_info:
- fix crash when extension version is 0.
- add raw ret value for extensions

postgresql_ext (i put it in this PR because it needs for integration tests):
- fix extension of version 0 handling

##### ISSUE TYPE

- Bugfix Pull Request

##### COMPONENT NAME
plugins/modules/postgresql_ext.py
plugins/modules/postgresql_info.py

### CODE REVIEW

**Comment by Andersson007** [comment]
cc @phemmer @tcraxs @kostiantyn-nemchenko @hunleyd

File: changelogs/fragments/138-postgresql_info_fix_ver_handling.yml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+bugfixes:
+- postgresql_info - Fix extension version handling when it has 0 value (https://github.com/ansible-collections/community.postgresql/issues/137).
+- postgresql_ext - Fix extension version handling when it has 0 value (https://github.com/ansible-collections/community.postgresql/pull/138).
```
Comment (tcraxs) [inline changelogs/fragments/138-postgresql_info_fix_ver_handling.yml:3]: Is the linking intended of issue and pull request?
I think #136 is the issue for `postgresql_ext` and #137 for `postgresql_info`

```suggestion
- postgresql_info - Fix extension version handling when it has 0 value (https://github.com/ansible-collections/community.postgresql/issues/137).
- postgresql_ext - Fix extension version handling when it has 0 value (https://github.com/ansible-collections/community.postgresql/issues/136).
```

**Review by tcraxs — APPROVED** [review]
Just a small change/hint in the changelog, other parts LGTM. Thanks @Andersson007

File: changelogs/fragments/138-postgresql_info_fix_ver_handling.yml (hunk 1 above)
Comment (Andersson007) [reply changelogs/fragments/138-postgresql_info_fix_ver_handling.yml]: @tcraxs ah, yes, i somehow missed that there's the postgresql_ext related issue as well:) Committed, thanks!

**Review by tcraxs — APPROVED** [review]
LGTM

**Comment by Andersson007** [comment]
fyi: there's unrelated CI bug https://github.com/ansible/ansible/issues/75647, please ignore

**Comment by Andersson007** [comment]
@phemmer thanks for reporting the issue!
@tcraxs @hunleyd (i saw your thumbs up) thanks for reviewing!

**Comment by Andersson007** [comment]
`community.postgresql` 1.5.0 has been released and is available to install via the `ansible-galaxy` command.
It also contains these changes!

**Pull request merged**